Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -393,9 +394,9 @@ static void BPWritten(const BPCmd& bp)
     if (bp.changes & 12)
       PixelShaderManager::SetZTextureOpChanged();
 #if defined(_DEBUG) || defined(DEBUGFAST)
-    const char* pzop[] = {"DISABLE", "ADD", "REPLACE", "?"};
-    const char* pztype[] = {"Z8", "Z16", "Z24", "?"};
-    PRIM_LOG("ztex op=%s, type=%s", pzop[bpmem.ztex2.op], pztype[bpmem.ztex2.type]);
+    static constexpr const char* pzop[] = {"DISABLE", "ADD", "REPLACE", "?"};
+    static constexpr const char* pztype[] = {"Z8", "Z16", "Z24", "?"};
```

Comment:
Any particular reason you left those as `const char*[]` but changed the ones in `BPFunctions` to `std::string_view[]`?
Not poking you to change it (unless it makes sense to), I mostly wanna know if one has an advantage over the other, especially when they'd be considered equivalent.

Reply:
Just overlooked them. Changed these over as well.